testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -760,6 +719,47 @@ impl ClusterSwarm for ClusterSwarmKube {
             workspace
         ))
     }
+
+    async fn put_file(
+        &self,
+        node: &str,
+        pod_name: &str,
+        path: &str,
+        content: Vec<u8>,
+    ) -> Result<()> {
+        let bucket = "toro-cluster-test-flamegraphs";
+        let run_id = env::var("RUN_ID").expect("RUN_ID is not set.");
+        libra_retrier::retry_async(libra_retrier::fixed_retry_strategy(5000, 15), || {
+            let run_id = &run_id;
```

Comment:
u do not need a local variable for run_id

Reply:
What do you mean? I think this is intentional to make &run_id to move.

Reply:
u can either use clone()? but just styling preference, i think its ok.